src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -97,4 +98,16 @@ protected static Configuration getModuleConfig(String moduleName, String moduleI
         return getModuleConfig(CONFIGURATION, moduleName, moduleId);
     }
 
+    /**
+     * Returns a list of all {@link Configuration} instances for the given module IDs.
+     *
+     * @param moduleIds module IDs.
+     * @return List of {@link Configuration} instances.
+     * @throws CheckstyleException if there is an error with the config.
+     */
+    protected static List<Configuration> getModuleConfigsByIds(String... moduleIds)
```

Comment:
This wrapper method is new, as this class has the configuration for google. The heart of the code for this is in the base IT so it can be used by other configurations, like sun.